tests/unittests/test_dev_mode.py

```diff
@@ -0,0 +1,93 @@
+from distutils.command.config import config
+from genericpath import isfile
+import unittest
+import os, json
+from tap_deputy.client import DeputyClient
+from tap_deputy.utils import write_config
+from unittest import mock
+from singer.utils import strftime
+from singer.utils import now, strftime, strptime_to_utc
+from dateutil.parser import parse
+from datetime import timedelta
+
+
+test_config = {
+    "client_id": "client_id",
+    "client_secret": "client_secret",
+    "domiain": "domain",
+    "start_date": "2021-09-20T00:00:00Z",
+    "redirect_uri": "redirect_uri",
+    "refresh_token": "old_refresh_token",
+    "access_token": "old_access_token",
+    "expires_at": strftime(now())
+}
+test_config_path = "/tmp/test_config.json"
+
+def write_new_config_file():
+    with open(test_config_path, 'w') as config:
+       config.write(json.dumps(test_config))
+
+def tear_down():
+    if os.path.isfile(test_config_path):
+        os.remove(test_config_path)
+    
+    if os.path.isfile(test_config_path):
+        os.remove(test_config_path)
```

Comment:
Please use [unittest.TestCase.tearDown](https://docs.python.org/3/library/unittest.html#unittest.TestCase.tearDown)

Reply:
Added the [unittest.TestCase.tearDown](https://docs.python.org/3/library/unittest.html#unittest.TestCase.tearDown)